packages/design-system/src/components/TextField/TextField.js

```diff
@@ -51,6 +58,7 @@ const TextField = props => {
 
 TextField.propTypes = {
   labelText: PropTypes.string,
+  labelAdornment: PropTypes.node,
```

Comment:
Add a comment about what it is - your comment will appear in the docs the "Props & methods" section of TextField properties. 
Try to add it to the example of TextField.

Reply:
@kamilmateusiak in case of the example I was feeling a little bit confused as in the docs for `TextxField` component there are no dedicated examples for particular props so it would look like something very special 😕 the only example is for using your own input and that is all. But ofc I would add an explanation as a comment for the prop itself as you suggested 🙂 👍